src/libraries/Microsoft.WindowsDesktop.Internal.Transport/src/Microsoft.WindowsDesktop.Internal.Transport.proj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.Build.NoTargets">
+  <PropertyGroup>
+    <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
+    <IsShipping>false</IsShipping>
+    <!-- Reference the outputs to have them available as build outputs. -->
+    <NoTargetsDoNotReferenceOutputAssemblies>false</NoTargetsDoNotReferenceOutputAssemblies>
+    <IsPackable>true</IsPackable>
+    <IncludeBuildOutput>true</IncludeBuildOutput>
+    <DebugSymbols>false</DebugSymbols>
+    <DebugType>none</DebugType>
+    <!-- This is non-shipping package. -->
+    <DisablePackageBaselineValidation>true</DisablePackageBaselineValidation>
+    <PackageDescription>Internal transport package to provide windowsdesktop with the assemblies that make up the Microsoft.WindowsDesktop.App shared framework.</PackageDescription>
+    <!-- Reference elements are missing from the nuspec: https://github.com/NuGet/Home/issues/8684. -->
+    <NoWarn>$(NoWarn);NU5131</NoWarn>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <!-- Requires Private=true to calculate ReferenceCopyLocalPaths items. -->
+    <ProjectReference Include="@(WindowsDesktopCoreAppLibrary->'$(LibrariesProjectRoot)%(Identity)\src\%(Identity).csproj')" PrivateAssets="all" Pack="true" Private="true" IncludeReferenceAssemblyInPackage="true" />
```

Comment:
just to understand fully, could you explain why we reference the src and not the ref project?

Reply:
https://github.com/dotnet/runtime/pull/57239 this might provide more context, i think the ref project references will be added transitively

Reply:
There are two reasons why reference project aren't directly referenced:
1. Technical reason: The transport package contains both reference assemblies and implementation assemblies. The `ResolveAssemblyReference` task doesn't under their difference as they share the same identity and assembly version and hence one of them is removed. Instead, as source assemblies already have the `ReferenceAssembly` metadata on them which points to the reference assembly, using that to include them.
2. Design reason: Reference assemblies are just a by-product of the identity's build. It doesn't matter if it comes directly from the csc invocation of the source project or via a P2P from the reference project. We should always treat them as a by-product of the identity's build. I myself treat source projects as the only identity and try to hide reference projects from my thinking graph. Not taking a dependency on reference projects also allows to converge them in the near future.